qubes-castable/src/lib.rs

```diff
@@ -411,44 +396,40 @@ macro_rules! castable {
         $p struct $s {
             $(
                 $(#[doc = $n])*
-                pub $name : $ty,
-            )*
+                pub $name : $ty
+            ),*
         }
         // SAFETY:
         //
-        // - The first static_assert! checks that the size of each member is
-        //   equal to the correspoding SIZE constant.  It also checks that
-        //   each member implements Castable.
-        // - The second static_assert! checks that the size of the struct is
-        //   equal to the sum of the sizes of its members.  This means that
-        //   the struct cannot have any padding.
+        // The static_assert! in the Default::default() implementation checks
```

Comment:
You might want to make `static_assert!` use `const _` instead of `let _`, such that it can also be used outside of functions. Then you wouldn't need to piggy-back on `Default::default` just for its function context.